File: HovyFridge/app/src/main/java/com/niklyadov/hovyfridge/data/entities/VersionInfo.kt
package com.niklyadov.hovyfridge.data.entities

data class VersionInfo(
    val id : String,
    val majorNumber: Int,
    val minorNumber: Int,
    val name: String,
    val downloadUrl : String
)

fun VersionInfo.isGreaterThan(versionInfo: VersionInfo)
        = majorNumber >=  versionInfo.majorNumber && minorNumber > versionInfo.minorNumber

File: HovyFridge.Web/HovyFridge.Api/Data/Entity/VersionInfo.cs
namespace HovyFridge.Api.Data.Entity
{
    public class VersionInfo
    {
        public Guid Id { get; set; }
        public int MajorNumber { get; set; }
        public int MinorNumber { get; set; }
        public string Name { get; set; } = "Undefined";
        public string DownloadUrl { get; set; }
    }
}